internal/sveltos/profile.go

```diff
@@ -261,35 +249,6 @@ func GetSpec(opts *ReconcileProfileOpts) (*sveltosv1beta1.Spec, error) {
 		})
 	}
 
-	for _, hc := range opts.HelmChartOpts {
-		helmChart := sveltosv1beta1.HelmChart{
-			RepositoryURL:    hc.RepositoryURL,
-			RepositoryName:   hc.RepositoryName,
-			ChartName:        hc.ChartName,
-			ChartVersion:     hc.ChartVersion,
-			ReleaseName:      hc.ReleaseName,
-			ReleaseNamespace: hc.ReleaseNamespace,
-			HelmChartAction:  sveltosv1beta1.HelmChartActionInstall,
-			RegistryCredentialsConfig: &sveltosv1beta1.RegistryCredentialsConfig{
-				PlainHTTP:             hc.PlainHTTP,
-				InsecureSkipTLSVerify: hc.InsecureSkipTLSVerify,
-				CredentialsSecretRef:  hc.CredentialsSecretRef,
-			},
-		}
-
-		if hc.PlainHTTP {
-			// InsecureSkipTLSVerify is redundant in this case.
-			// At the time of implementation, Sveltos would return an error when PlainHTTP
-			// and InsecureSkipTLSVerify were both set, so verify before removing.
-			helmChart.RegistryCredentialsConfig.InsecureSkipTLSVerify = false
-		}
```

Comment:
i haven't noticed it from the first glance, could you add such a statement to the `GetHelmCharts` function as well, somewhere after the 

```go
opt := sveltosv1beta1.HelmChart{ // opt -> helmChart?
// ... 
```

Reply:
done